src/PAModelTests/YamlTest.cs

```diff
@@ -109,6 +109,48 @@ public void NewLinesRoundtrip(string value)
             Assert.AreEqual(normalizedValue, NormNewlines(p.Value));
         }
 
+
+        [DataTestMethod]
+        [DataRow(true)]
+        [DataRow(false)]
+        public void WriteBool(bool value)
+        {
+            var sw = new StringWriter();
+            var yw = new YamlWriter(sw);
+            yw.WriteProperty("P0", value);
+
+            var t = sw.ToString();
+            var expected = $@"P0: ={value.ToString().ToLower()}
+";
+            Assert.AreEqual(expected.Replace("\r\n", "\n"), t.Replace("\r\n", "\n"));
+        }
+
+        [TestMethod]
+        public void WriteInt()
+        {
+            var sw = new StringWriter();
+            var yw = new YamlWriter(sw);
+            yw.WriteProperty("P0", 12);
+
+            var t = sw.ToString();
+            var expected = @"P0: =12
+";
+            Assert.AreEqual(expected.Replace("\r\n", "\n"), t.Replace("\r\n", "\n"));
+        }
+
+        [TestMethod]
+        public void WriteDouble()
```

Comment:
Could we test all three types in one test with data driven testing?

Reply:
As long as these are separate public methods, I'd prefer they have separate unit tests methods as well.